core/src/main/java/com/graphhopper/reader/osm/OSMReader.java

```diff
@@ -346,6 +355,26 @@ protected void processWay(ReaderWay way) {
             }
         }
 
+        List<CustomArea> customAreas = estimatedCenter == null ? emptyList() : customAreaIndex.query(estimatedCenter.lat, estimatedCenter.lon);
+        // special handling for countries: since they are built-in with GraphHopper they are always fed to the
+        // encodingmanager
+        NewCountry country = null;
```

Comment:
If we later substitute `null` with `NewCountry.MISSING`, i'd rather use it as the initial value.

Reply:
Yes, I probably forgot updating this when I added the `MISSING` value.